### PR TITLE
Freeze config files for processing on compute canada: graham

### PR DESCRIPTION
**Pull request by PaulBautin**
Freeze config files for processing on compute canada: graham

DONE:
- remove useless config parameters
- set config parameters

### CODE REVIEW

File: job_template.sh [hunk 1]
```diff
@@ -4,5 +4,7 @@
 #SBATCH --cpus-per-task=32    # number of OpenMP processes
 #SBATCH --mem=128G
 #SBATCH --mail-user=paul.bautin@polymtl.ca
+#SBATCH --mail-type=ALL
+#SBATCH --mail-type=END
```
Comment (jcohenadad) [inline job_template.sh:8]: not needed (included in "#SBATCH --mail-type=ALL")

Comment (PaulBautin) [reply job_template.sh]: Right, implemented in https://github.com/sct-pipeline/csa-atrophy/pull/73/commits/424971f5ecd8f1940bcc1d11b4cc38aef9c10a3c

**Review by jcohenadad — APPROVED** [review]

**Comment by PaulBautin** [comment]
@jcohenadad, should we keep this PR as testing branch or do we merge it for first release/example?

**Comment by jcohenadad** [comment]
we can merge

**Pull request merged**